kolibri/utils/options.py

```diff
@@ -271,6 +273,22 @@ def multiprocess_bool(value):
         return False
 
 
+def inherits_from_storage(value):
+    try:
+        modules = value.split(".")
+        klass = modules.pop()
+        module_path = ".".join(modules)
+        module = importlib.import_module(module_path)
```

Comment:
Note that Django exposes a utility called `import_string` which we already use in this module for loading classes by a string dot path, so this seems preferable to use here.